.changeset/sixty-carrots-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': patch
+---
+
+Disable 'Create' button in template after initial click to avoid confusion when there is slowness in template execution
```

Comment:
now accidentally we have two files here :) but apart from that it looks great!

Reply:
Removed this one